cql/CollateManagementData.cql

```diff
@@ -387,6 +387,27 @@ define Cin2orCin3Biopsies:
 define MostRecentCin2orCin3Biopsy:
   First(Cin2orCin3Biopsies)
 
+define Cin3Biopsies:
+  SortedBiopsyReports S
+    where S.riskTableInput in {'CIN3'}
+
+define MostRecentCin3Biopsy:
+  First(Cin3Biopsies)
+      
+define Cin2Biopsies:
+  SortedBiopsyReports S
+    where S.riskTableInput in {'CIN2'}
+
+define MostRecentCin2Biopsy:
+  First(Cin2Biopsies)
+  
+define HsilBiopsies:
+  SortedBiopsyReports S
+    where S.riskTableInput in {'HSIL'}
+
+define MostRecentHsilBiopsy:
+  First(HsilBiopsies)
```

Comment:
What are these expressions supposed to be doing? Are they looking for the most recent "Hsil, Unspecified" biopsy? There is no `riskTableInput` in the [BiopsyInterpretation function](https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/blob/cin2-surveillance/cql/DashboardLibrary.cql#L210-L226) that matches with 'HSIL'.

If that is the case, the name of these expressions should also be changed from Hsil to UnspecifiedHsil.

Reply:
@mrnosal Any thoughts on this? This comment refers to `HsilBiopsies` and `MostRecentHsilBiopsy`.

Reply:
You're right about the 'HSIL, Unspecified'. 798efb6 fixes this.